middlewares/logger.js

```diff
@@ -0,0 +1,31 @@
+const Student = require('../models/student.model');
+const Tutor = require('../models/tutor.model');
+const jwt = require('jsonwebtoken');
+
+const logger = async (req, res, next) => {
+    try {
+        const { email, password } = req.body;
+        let user;
+        const student = await Student.authenticate(email, password);
+        if (student) {
```

Comment:
You can refactor a little more

Reply:
Hello Leonel, I've done another refactor, I think now it's much cleaner